apps/web/src/components/HolidayAlert/HolidayAlert.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { useTheme } from "@gc-digital-talent/theme";
+import { AlertImage } from "@gc-digital-talent/ui";
+
+import lightImage from "~/assets/img/Holiday_2023_graphic_Light_mode.webp";
```

Comment:
the image itself isn't in any mode. uppercase file name can cause issues so better to avoid them altogether.
```suggestion
import lightImage from "~/assets/img/holiday_2023_graphic_light.webp";
```

Reply:
Renamed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8769/commits/f2cbdb70c4120b7feb65cdfb41e06567ccaa0956